src/HTSPDemuxer.cpp

```diff
@@ -454,82 +458,80 @@ void CHTSPDemuxer::ParseSubscriptionStart ( htsmsg_t *m )
 
     /* Find stream */
     m_streamStat[idx] = 0;
+    PVR_STREAM_PROPERTIES::PVR_STREAM stream;
```

Comment:
You need to zero memset <code>stream</code>, or is it guaranteed that every member will be set in every iteration? Even if this is the case, I would memset to be on the safe side, for instance if the struct gets extended in the future.